web/public/mgmt/config.yml

```diff
@@ -3027,7 +3035,15 @@ collections:
         widget: "select"
         multiple: false
         required: true
-        options: ["loop.svg", "capitol.svg"]
+        options:
+          - { label: "Update", value: "update-purple.svg" }
+          - { label: "Government", value: "govt-purple.svg" }
+          - { label: "Add", value: "add-purple.svg" }
+          - { label: "Change", value: "change-purple.svg" }
+          - { label: "Sell", value: "sell-purple.svg" }
+          - { label: "Close", value: "close-purple.svg" }
+          - { label: "Edit", value: "edit-purple.svg" }
+          - { label: "File", value: "file-purple.svg" }
       - { label: "External Link", name: "externalRoute", widget: "nospace", required: true }
```

Comment:
<img width="563" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/faa4f877-85fa-4418-a865-bc090dc1bdec">
This was how I was thinking about things initially, I think this is fine and not wrong (espeicially if it only happens twice) but I think a slightly more centeralized and robust solution is probably a bit better.

Reply:
dust - It might be a good idea to create an uneditable mapping collection, the industry mapping ticket I'm working on also prevents content from editing the field